apps/web/src/app/vault/individual-vault/vault-onboarding/services/vault-onboarding.service.ts

```diff
@@ -26,20 +25,17 @@ const VAULT_ONBOARDING_KEY = new UserKeyDefinition<VaultOnboardingTasks>(
     clearOn: [], // do not clear tutorials
   },
 );
-
 @Injectable()
 export class VaultOnboardingService implements VaultOnboardingServiceAbstraction {
-  private vaultOnboardingState: ActiveUserState<VaultOnboardingTasks>;
-  vaultOnboardingState$: Observable<VaultOnboardingTasks>;
+  vaultOnboardingTasksState: SingleUserState<VaultOnboardingTasks> | null = null;
```

Comment:
can we have a private method for this instead for we avoid storing state in a class property
```suggestion
  private vaultOnboardingState(userId: UserId): SingleUserState<VaultOnboardingTasks> {
    return this.stateProvider.getUser(userId, VAULT_ONBOARDING_KEY);
  }
```